HttpCache.php

```diff
@@ -83,6 +83,8 @@
 
     public function __construct(HttpKernelInterface $kernel, $cacheDir = null)
     {
+        @trigger_error('The '.__CLASS__.' class is deprecated since version 1.2 and will be removed in 2.0. Use FOS\HttpCacheBundle\SymfonyCache\EventDispatchingHttpCache instead.', E_USER_DEPRECATED);
```

Comment:
When the whole class is deprecated, you could also put this above the class definition directly.